mmv1/templates/terraform/examples/iam_deny_policy_basic.tf.erb

```diff
@@ -0,0 +1,44 @@
+resource "google_project" "project" {
+  provider        = google-beta
+  project_id      = "tf-test%{random_suffix}"
+  name            = "tf-test%{random_suffix}"
+  org_id          = "<%= ctx[:test_env_vars]['org_id'] %>"
+  billing_account = "<%= ctx[:test_env_vars]['billing_account'] %>"
+}
+
+resource "google_iam_deny_policy" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  parent   = urlencode("cloudresourcemanager.googleapis.com/projects/${google_project.project.project_id}")
```

Comment:
No action: Oh, frustrating! They can't use the https://google.aip.dev/122#full-resource-names format because it's getting put into a URL.